apps/fluent-tester/src/FluentTester.tsx

```diff
@@ -178,9 +179,7 @@ export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props:
                     }
                   }}
                   style={mobileStyles.testListItem}
-                  testID={description.testPageButton}
-                  // Please read http://93days.me/testing-react-native-application/ to understand why we set accessibilityLabel here.
```

Comment:
I know the testID and accessibilityLabel are the same here, but I'd still suggest leaving them separated here. It looks like there's an important comment related to accessibilityLabel. We should keep that